src/components/checkbox/Checkbox.scss

```diff
@@ -4,6 +4,12 @@
  * Checkbox
  **************************************/
 
+.Checkbox--isFocused {
```

Comment:
for SUIT naming, this would probably be `.bdl-Checkbox-input.bdl-is-focused`:
- `bdl-Checkbox-input` since the class would be on the descendant element
- `bdl-is-focused` as state should be a separate class

Reply:
Ah, I wondered about this. Thank you very much for giving the answer there 👍